src/App.jsx

```diff
@@ -44,56 +84,55 @@ function App() {
         <Route path="favorite" element={<FavoritePage />} />
-        <Route
-          path="account"
-          element={<div style={{ marginBottom: '500px' }}>Акаунт</div>}
-        />
         <Route
           path="cart"
           element={<div style={{ marginBottom: '500px' }}>Корзина</div>}
         />
         <Route path="conditions" element={<ConditionsPage />} />
         <Route path="policy" element={<PrivacyPolicyPage />} />
-      </Route>
 
-      <Route path="crm" element={<LayoutCRM />}>
-        <Route
-          path="dashbord"
-          element={<div style={{ marginBottom: '500px' }}>Dashboard page</div>}
-        />
-        <Route
-          path="orders"
-          element={<div style={{ marginBottom: '500px' }}>Order page</div>}
-        />
-        <Route
-          path="orders/:params"
-          element={<div style={{ marginBottom: '500px' }}>orders params</div>}
-        />
-        <Route path="products" element={<CrmProductsPage />} />
+        <Route path="crm" element={<LayoutCRM />}>
+          <Route
+            path="dashbord"
+            element={
+              <div style={{ marginBottom: '500px' }}>Dashboard page</div>
+            }
+          />
+          <Route
+            path="orders"
+            element={<div style={{ marginBottom: '500px' }}>Order page</div>}
+          />
+          <Route
+            path="orders/:params"
+            element={<div style={{ marginBottom: '500px' }}>orders params</div>}
+          />
+          <Route path="products" element={<CrmProductsPage />} />
 
-        <Route path="products/:params" element={<CrmAddNewProduct />} />
+          <Route path="products/:params" element={<CrmAddNewProduct />} />
 
-        <Route
-          path="clients"
-          element={<div style={{ marginBottom: '500px' }}>Clients page</div>}
-        />
-        <Route
-          path="clients/:params"
-          element={<div style={{ marginBottom: '500px' }}>clients params</div>}
-        />
-        <Route
-          path="opinions"
-          element={<div style={{ marginBottom: '500px' }}>Opinions Page</div>}
-        />
-        <Route path="settings" element={<CrmSettingsPage />} />
-        <Route index element={<Navigate to="dashbord" replace />} />
-      </Route>
-    </Routes>
+          <Route
+            path="clients"
+            element={<div style={{ marginBottom: '500px' }}>Clients page</div>}
+          />
+          <Route
+            path="clients/:params"
+            element={
+              <div style={{ marginBottom: '500px' }}>clients params</div>
+            }
+          />
+          <Route
+            path="opinions"
+            element={<div style={{ marginBottom: '500px' }}>Opinions Page</div>}
+          />
+          <Route path="settings" element={<CrmSettingsPage />} />
+          <Route index element={<Navigate to="dashbord" replace />} />
+        </Route>
+      </Routes>
+      <ModalPortal>
+        <Auth />
+      </ModalPortal>
```

Comment:
https://blog.stackademic.com/practical-example-creating-a-modal-component-using-react-portals-91325c7cb0f0

good example of using portals for modal